build/DevAnalyzers.props

```diff
@@ -1,4 +1,10 @@
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="Toolchain for run Generator on Linux" Condition="'$(PkgMicrosoft_Net_Compilers)'=='' And $([MSBuild]::IsOSPlatform('Linux'))">
```

Comment:
On my Ubuntu 22.04.1, when compiling on rider, the build fails because the builder can run. The build log shows a warning explaining that the generator and analyzer cannot be instantiated because the assembly 'Microsoft.CodeAnalysis, Version = 4.1.0.0, Culture = neutral, PublicKeyToken = 31bf3856ad364e35' could not be found.



To fix the problem I had to add the reference to "Microsoft.Net.Compilers".

In particular, the line is used to prevent the double import when referencing both the DevAnalyzer and the DevGenerator

Reply:
This is because the version of .NET SDK you installed is too old.

Reply:
I'll try